docs/strategies/retry.md

```diff
@@ -104,7 +104,44 @@ new ResiliencePipelineBuilder().AddRetry(new RetryStrategyOptions
 | `OnRetry`          | `null`                                                                     | Action executed when retry occurs.                                                       |
 | `MaxDelay`         | `null`                                                                     | Caps the calculated retry delay to a specified maximum duration.                         |
 
-## Patterns and anti-patterns
+## Patterns
+
+### Limiting the maximum delay
+
+In some cases, you might want to set a limit on the calculated delay. This is beneficial when multiple retries are anticipated, and you wish to prevent excessive wait times between these retries.
+
+Consider the following example of a long-running background job:
+
+<!-- snippet: retry-pattern-max-delay -->
+```cs
+ResiliencePipeline pipeline = new ResiliencePipelineBuilder()
+    .AddRetry(new()
+    {
+        Delay = TimeSpan.FromSeconds(2),
+        MaxRetryAttempts = int.MaxValue,
+
+        // Initially, we aim for an exponential backoff, but after a certain number of retries, we set a maximum delay of 15 minutes.
```

Comment:
Would it make sense to explicitly use `BackoffType = DelayBackoffType.Exponential` ?